src/py42/clients/settings/device_settings.py

```diff
@@ -168,9 +174,9 @@ def __repr__(self):
 class BackupSet(UserDict, object):
     """Helper class for managing device backup sets and Org device default backup sets."""
 
-    def __init__(self, settings_manager, changes_dict, backup_set_dict):
+    def __init__(self, settings_manager, backup_set_dict):
         self._manager = settings_manager
-        self._changes = changes_dict
+        self._changes = settings_manager.changes
```

Comment:
This change wasn't related to the bug, just made constructing the BackupSet object a bit simpler.